oneview/data_source_server_profile_template.go

```diff
@@ -609,11 +1026,22 @@ func dataSourceServerProfileTemplateRead(d *schema.ResourceData, meta interface{
 	}
 	boot := make([]map[string]interface{}, 0, 1)
 	boot = append(boot, map[string]interface{}{
-		"manage_boot": spt.Boot.ManageBoot,
-		"boot_order":  bootOrder,
+		"compliance_control": spt.Boot.ComplianceControl,
+		"manage_boot":        spt.Boot.ManageBoot,
+		"boot_order":         bootOrder,
 	})
 	d.Set("boot", boot)
 
+	bootModeOptions := make([]map[string]interface{}, 0, 1)
+	bootModeOptions = append(bootModeOptions, map[string]interface{}{
+		"manage_mode":     spt.BootMode.ManageMode,
+		"mode":            spt.BootMode.Mode,
+		"pxe_boot_policy": spt.BootMode.PXEBootPolicy.String(),
+		"secure_boot":     spt.BootMode.SecureBoot,
+	})
+
+	d.Set("boot_mode", bootModeOptions)
+
 	overriddenSettings := make([]interface{}, 0, len(spt.Bios.OverriddenSettings))
```

Comment:
I think we can put overriddenSettings  in ```if spt.Bios != nil``` because if there is no Bios Settings, terraform will crash.

Reply:
done